app/routes.py

```diff
@@ -1,2 +1,231 @@
+        videos = Video.query.all()
+        videos_response = []
+        for one_video in videos:
+            videos_response.append(one_video.to_json_video())
+        
+        return jsonify(videos_response)
+    elif request.method == "POST":
+        request_body = request.get_json()
+        if "title" not in request_body or "release_date" not in request_body or "total_inventory" not in request_body:
+            return make_response ({"details": "Invalid data"}),400
+        else:
+            new_video= Video(title=request_body["title"],
+                    release_date=request_body["release_date"],
+                    total_inventory=request_body["total_inventory"],
+                    available_inventory=request_body["total_inventory"])
+        db.session.add(new_video)
+        db.session.commit()
+
+# they want us to return the id only 
+        return make_response({"id": new_video.video_id}), 201 
+
+
+
+@videos_bp.route("/<video_id>", methods=["GET", "PUT","DELETE"])
+def get_one_video(video_id):
+    video = Video.query.get(video_id)
+
+    if request.method == "GET":
+        if video == None:
+            return make_response("",404)
+        else:
+            return make_response(video.to_json_video()), 200
+
+    elif request.method == "PUT":
+        request_body = request.get_json()
+        if video == None:
+            return make_response("",404)
+        elif "title" not in request_body or "release_date" not in request_body or "total_inventory" not in request_body:
+            return make_response ({"details": "Invalid data"}),400    
+        # converting postman json to a dict table
+        else:
+            form_data = request.get_json()
+
+            video.title = form_data["title"]
+            video.release_date = form_data["release_date"]
+            video.total_inventory = form_data["total_inventory"]
+
+            db.session.commit()
+            return make_response(video.to_json_video()), 200
+    
+    elif request.method == "DELETE":
+        if video == None:
+            return make_response("",404)        
+        else:
+            db.session.delete(video)
+            db.session.commit()
+            return make_response ({"id": video.video_id}), 200
+
+# This route will check out a video to a customer, and update the data in the database 
+@rental_bp.route("/check-out", methods=["POST"])
+def checkout_video():
+    request_body = request.get_json()
+
+    # Required Request Body Parameters
+    customer = Customer.query.get(request_body["customer_id"])
+    video = Video.query.get(request_body["video_id"])
+    # check if customer id and video id are int
+    if type(customer) != int or type(video) != int:
+        return jsonify({"details":"Invalid data"}, 404)    
+    # return back detailed errors/ status 404: Not Found if the customer or video does not exist
+    if customer == None or video == None:
+        return jsonify({"details":"Invalid data"}, 404)
```

Comment:
We should not pass 404 into jsonify. There are several different ways to build this response:
`return {"details":"Invalid data"}, 404`
`return jsonify({"details":"Invalid data"}), 404`
`return make_response({"details":"Invalid data"}, 404)`

Reply:
This comment applies to other places where the response code is included as an argument passed to jsonify